velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -46,58 +44,6 @@ core::AggregationNode::Step toAggregationStep(
   }
 }
 } // namespace
-namespace {
-template <TypeKind KIND>
-VectorPtr setVectorFromVariantsByKind(
```

Comment:
Is this just a refactor to move this code in a separate .h/.cpp files? Is it necessary for this PR? If not, would you extract it into a separate PR?

Reply:
I extracted it into a separate PR here. https://github.com/facebookincubator/velox/pull/2857 And i will rebase this PR based on it later.